### ISSUE TITLE
Not able to query for nested relations using dgraph-orm

### ISSUE THREAD

**Issue by PraveenKumarPalai** [ashokvishwakarma/dgraph-orm#16]
I am using **dgraph-orm for fetching nested relational values** but it works for single level but not multiple level.
I am getting the page details but **unable to fetch the avatar** of the page.

Here is my snippet:

```
let posts = await PagePost.has('page_id', {
      filter: {
        page_id: {
          uid_in: [page_id]
        }
      },
      include: {
        page_id: {
          as: 'page',
          include: {
            avatar: {
              as: 'avatar'
            }
          }
        },
        owner_id: {
          as: 'postedBy'
        }
      },
      order: [], // accepts order like the above example
      first: perPage, // accepts first
      offset: offset, // accepts offset
    });
```

I am not getting avatar for the attribute page_id:

```
 {
            "uid": "0x75b4",
            "title": "",
            "content": "haha",
            "created_at": "2019-09-23T08:50:52.957Z",
            "status": true,
            "page": [
                {
                    "uid": "0x75ac",
                    "name": "Saregamaapaaaa...",
                    "description": "This a is place where you can listen ti thrilling music.",
                    "created_at": "2019-09-23T06:46:50.756Z",
                    "status": true
                }
            ],
            "postedBy": [
                {
                    "uid": "0x3",
                    "first_name": "Mohit",
                    "last_name": "Talwar",
                    "created_at": "2019-07-11T11:37:33.853Z",
                    "status": true
                }
            ]
        }
```

***Is there a support for multilevel field querying in the orm??***

**Comment by ashokvishwakarma** [comment]
There was some issue with ORM itself it was not able to recognize the correct model name for multilevel includes and generating the wrong queries.

Fixed the same in version 1.2.4, please run `npm update dgraph-orm --save` to update your DgraphORM.

Thanks for the issue.

**Comment by PraveenKumarPalai** [comment]
This update works for simple queries but  when there are multiple relations then it fails to update the correct model for each relation.

Here's the query that is being logged:

```
{
      page_post (func: has(page_post.page_id), first: 25)  @filter(uid_in(page_post.page_id, 0x30d4f)) {
        uid
title: page_post.title
content: page_post.content
created_at: page_post.created_at
updated_at: page_post.updated_at
status: page_post.status
        location: page_post.location{
        uid
lat: location.lat
long: location.long
address: location.address
locality: location.locality
city: location.city
state: location.state
country: location.country
pincode: location.pincode
created_at: location.created_at
status: location.status
        
      }banner: location.banner{
        uid
name: media.name
real_name: media.real_name
path: media.path
object_type: media.object_type
object_id: media.object_id
media_size: media.media_size
extension: media.extension
created_at: media.created_at
status: media.status
        
      }page: media.page_id{
        uid
name: page.name
description: page.description
created_at: page.created_at
status: page.status
        avatar: page.avatar{
        uid
name: media.name
real_name: media.real_name
path: media.path
object_type: media.object_type
object_id: media.object_id
media_size: media.media_size
extension: media.extension
created_at: media.created_at
status: media.status
        
      }
      }postedBy: page.owner_id{
        uid
first_name: user.first_name
last_name: user.last_name
email: user.email
user_name: user.user_name
mobile: user.mobile
gender: user.gender
birthday: user.birthday
avatar: user.avatar
cover_image: user.cover_image
city: user.city
school: user.school
work: user.work
country: user.country
otp: user.otp
v_code: user.v_code
r_code: user.r_code
fb_id: user.fb_id
google_id: user.google_id
created_at: user.created_at
status: user.status
        
      }media: user.media{
        uid
name: media.name
real_name: media.real_name
path: media.path
object_type: media.object_type
object_id: media.object_id
media_size: media.media_size
extension: media.extension
created_at: media.created_at
status: media.status
        
      }
      }
    }
```
